apps/api/src/app/subscribers/e2e/update-subscriber.e2e.ts

```diff
@@ -72,13 +72,11 @@ describe('Update Subscriber - /subscribers/:subscriberId (PUT)', function () {
     );
 
     const response = await axiosInstance.put(
-      `${session.serverUrl}/v1/subscribers/123`,
+      `${session.serverUrl}/v1/subscribers/123/channels`,
```

Comment:
Let's call this endpoint `${session.serverUrl}/v1/subscribers/123/credentials` I Think it might be clearer indicating the purposes of this endpoint. As it's responsibility is mainly to set the credentials for a specific channel and not the other way around.

@ainouzgali @davidsoderberg any thoughts on the REST endpoint naming? Let me know if you need more context

Reply:
Agree as the /credentials explain better the update, and the /channels the data entity which I was aiming to reflect.